### ISSUE TITLE
cannot use *shell.PubSubRecord as type pubsub.Record

### ISSUE THREAD

**Issue by camstuart** [keks/go-orbitdb#5]
During a `go get` I get the error:

```
pubsub/ipfs-api/pubsub.go:15:2: shell.PubSubRecord.Data is a field, not a method
pubsub/ipfs-api/pubsub.go:15:2: cannot use *shell.PubSubRecord as type pubsub.Record in return argument:
        *shell.PubSubRecord does not implement pubsub.Record (missing Data method)
```
Generated by this line:

https://github.com/keks/go-orbitdb/blob/3164f35afc1a618f3b64e7baa16171ff4944689a/pubsub/ipfs-api/pubsub.go#L15

Is it not:
```Golang
return s.Next()
``` 
instead?

**Comment by chiragg6** [comment]
Hi @camstuart , I am facing the same issue. Were you able to find a fix to this?

It looks like the project is deserted. But maybe if @keks is interested in helping a bit, we could spin this into active development.

**Comment by keks** [comment]
@camstuart No, that wouldn't make much sense because it would just recurse infinitely.

I have the hunch that the API of go-ipfs-shell changed since I wrote this package. Unfortunately I currently don't really have time to update it to use the new one. I would be happy to merge PRs, though.

**Comment by camstuart** [comment]
I'm afraid I didn't solve this.
